tests/integration/transactions/test_xchain_claim.py

```diff
@@ -0,0 +1,114 @@
+from tests.integration.integration_test_case import IntegrationTestCase
+from tests.integration.it_utils import (
+    sign_and_reliable_submission_async,
+    test_async_and_sync,
+)
+from tests.integration.reusable_values import BRIDGE, DESTINATION, WITNESS_WALLET
+from xrpl.core.binarycodec import encode
+from xrpl.core.keypairs import sign
+from xrpl.models import (
+    AccountInfo,
+    Tx,
+    XChainAddClaimAttestation,
+    XChainClaim,
+    XChainCreateClaimID,
+)
+from xrpl.utils import xrp_to_drops
+from xrpl.wallet import Wallet
+
+
+class TestXChainClaim(IntegrationTestCase):
+    @test_async_and_sync(globals())
+    async def test_basic_functionality(self, client):
+        other_chain_source = Wallet.create().classic_address
+        claim_id_response = await sign_and_reliable_submission_async(
+            XChainCreateClaimID(
+                account=DESTINATION.classic_address,
+                xchain_bridge=BRIDGE.xchain_bridge,
+                signature_reward=BRIDGE.signature_reward,
+                other_chain_source=other_chain_source,
+            ),
+            DESTINATION,
+            client,
+        )
+        claim_id_hash = (
+            claim_id_response.result.get("tx_json") or claim_id_response.result
+        )["hash"]
+        claim_id_tx_response = await client.request(Tx(transaction=claim_id_hash))
+
+        nodes = claim_id_tx_response.result["meta"]["AffectedNodes"]
+        created_nodes = [
+            node["CreatedNode"] for node in nodes if "CreatedNode" in node.keys()
+        ]
+        claim_ids_ledger_entries = [
+            node
+            for node in created_nodes
+            if node["LedgerEntryType"] == "XChainOwnedClaimID"
+        ]
+        assert len(claim_ids_ledger_entries) == 1, len(claim_ids_ledger_entries)
+        xchain_claim_id = claim_ids_ledger_entries[0]["NewFields"]["XChainClaimID"]
```

Comment:
These variables are a bit hard to read because they all start with a long prefix. I'd recommend simplifying the names, and adding a comment near the top for `// Claim ID Data` so that the variables can be more easily sight-read. 

```suggestion
        hash = (
            claim_id_response.result.get("tx_json") or claim_id_response.result
        )["hash"]
        tx_response = await client.request(Tx(transaction=hash))

        nodes = tx_response.result["meta"]["AffectedNodes"]
        created_nodes = [
            node["CreatedNode"] for node in nodes if "CreatedNode" in node.keys()
        ]
        ledger_entries = [
            node
            for node in created_nodes
            if node["LedgerEntryType"] == "XChainOwnedClaimID"
        ]
        assert len(ledger_entries) == 1, len(ledger_entries)
        xchain_claim_id = ledger_entries[0]["NewFields"]["XChainClaimID"]
```

Reply:
I'd rather keep the variables more descriptive.